packages/elements-core/src/components/MosaicTableOfContents/TableOfContents.tsx

```diff
@@ -257,7 +257,7 @@ const Node = React.memo<{
       className="ElementsTableOfContentsItem"
     >
       <Item
-        id={getHtmlIdFromItemId(item.id)}
+        id={getHtmlIdFromItemId(item.slug || item.id)}
```

Comment:
Can you double check this still works with the scroll logic? It looks like it will: https://github.com/stoplightio/elements/blob/2b7f277fe401c7cb4644a46e4bf5e1c1ea698025/packages/elements-core/src/components/MosaicTableOfContents/TableOfContents.tsx#L39

Reply:
I did already, works :).